ftl/node/main.py

```diff
@@ -86,6 +85,14 @@
           instead of pushing to registry')
 
 parser.add_argument(
+    '--destination',
+    dest='destination_path',
+    action='store',
+    default="app",
```

Comment:
Should this be /app? Can you make the quoting consistent as well?

Reply:
done